.github/workflows/docker.yml

```diff
@@ -33,9 +33,20 @@ jobs:
           docker login docker.pkg.github.com -u BrewTestBot -p ${{secrets.GITHUB_TOKEN}}
           docker tag brew "docker.pkg.github.com/homebrew/brew/ubuntu${{matrix.version}}:$brew_version"
           docker push "docker.pkg.github.com/homebrew/brew/ubuntu${{matrix.version}}:$brew_version"
+          docker tag brew "docker.pkg.github.com/homebrew/brew/ubuntu${{matrix.version}}:latest"
+          docker push "docker.pkg.github.com/homebrew/brew/ubuntu${{matrix.version}}:latest"
       - name: Deploy the tagged Docker image to Docker Hub
         if: startsWith(github.ref, 'refs/tags/')
         run: |
           docker login -u brewtestbot -p ${{secrets.DOCKER_TOKEN}}
           docker tag brew "homebrew/ubuntu${{matrix.version}}:$brew_version"
           docker push "homebrew/ubuntu${{matrix.version}}:$brew_version"
+          docker tag brew "homebrew/ubuntu${{matrix.version}}:latest"
+          docker push "homebrew/ubuntu${{matrix.version}}:latest"
+      - name: Deploy the homebrew/brew Docker image to GitHub and Docker Hub
+        if: startsWith(github.ref, 'refs/tags/') && matrix.version == '20.04'
+        run: |
+            docker tag brew "docker.pkg.github.com/homebrew/brew/brew:latest"
+            docker push "docker.pkg.github.com/homebrew/brew/brew:latest"
+            docker tag brew "docker.pkg.github.com/homebrew/brew/brew:latest"
+            docker push "docker.pkg.github.com/homebrew/brew/brew:latest"
```

Comment:
Is one ment to be `$brew_version` or Docker Hub?

Reply:
🙈 Thanks for catching this error, Bo!